configure.ac

```diff
@@ -46,7 +44,27 @@ AC_ARG_WITH(extra-lib,
                           prefix where extra libraries are installed],
 [AC_SUBST(LDFLAGS, "$LDFLAGS -L${withval}")])
 
-CFLAGS_COMMON="-std=c++14 -Wall -fPIC -Wno-write-strings -I/usr/include/libnl3 -I/usr/include/swss"
+AC_ARG_WITH(extra-usr-lib,
+[  --with-extra-usr-lib=DIR
+                          prefix where extra libraries are installed],
+[AC_SUBST(LDFLAGS, "$LDFLAGS -L${withval}")])
+
+AC_CHECK_LIB([nl-3], [nl_addr_alloc])
```

Comment:
>nl-3 [](http://example.com/codeflow?start=14&length=4)

If you have installed Debian vanilla libnl3, and extract patched libnl3 into a folder, which one will you prefer?

Reply:
the extracted one, always. The '-L' includes the path to the extracted libs so it will be the ones that will be linked.